common/string_calls.h

```diff
@@ -154,6 +154,7 @@ int      g_strncmp_d(const char *c1, const char *c2, const char delim, int len);
 int      g_strcasecmp(const char *c1, const char *c2);
 int      g_strncasecmp(const char *c1, const char *c2, int len);
 int      g_atoi(const char *str);
+int      g_atoix(const char *str);
```

Comment:
Suggest adding a Doxygen-compatible function comment here rather than in `string_calls.c`. Here's an example. Feel free to change it:-

```c
/**
 * As `g_atoi()`, but also allows hexadecimal numbers
 *
 * Prefix hexadecimal numbers with '0x' 
 * 
 * @param str String to convert to an integer
 * @return Integer representation of string
 */
int      g_atoix(const char *str);
```

I'm hopeful at some stage we'll start to use Doxygen to read the source code and give us some useful documentation.

Reply:
thanks, @matt335672 
I'll add a Doxygen-compatible function comment.